### PR TITLE
test: switch on binlog test, part 1

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
switch on binlog test, part 1

### CODE REVIEW

File: cdc/entry/mounter.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func (m *Mounter) Mount(rawTxn model.RawTxn) (model.Txn, error) {
 			}
 			return t, nil
 		case *unknownKVEntry:
-			log.Warn("Found unknown kv entry", zap.Reflect("unknownKVEntry", e))
+			log.Debug("Found unknown kv entry", zap.Reflect("unknownKey", e.Key))
```
Comment (suzaku) [inline cdc/entry/mounter.go]: Use specific type field for `e.Key`?

File: tests/util/db.go [hunk 2]
```diff
@@ -118,6 +118,23 @@ func CheckSyncState(sourceDB, targetDB *sql.DB, schema string) bool {
...
+	sourceTableMap := make(map[string]struct{}, len(tables))
+	for _, table := range tables {
+		sourceTableMap[table] = struct{}{}
+	}
+	for _, table := range targetTables {
+		if _, exist := sourceTableMap[table]; !exist {
+			log.Print("The table in target db is not exist in source db:", table)
+			return false
+		}
+	}
```
Comment (suzaku) [inline tests/util/db.go]: Doesn't the diff check already check this for us?

Comment (leoppro) [reply tests/util/db.go]: when the table in target database is more than source database, the diff check will not return an error. This pr will fix this.

File: cdc/entry/mounter.go (hunk 1 above)
Comment (leoppro) [reply cdc/entry/mounter.go]: fixed

File: tests/util/db.go (hunk 2 above)
Comment (suzaku) [reply tests/util/db.go]: Add a comment to explain this?

Comment (leoppro) [reply tests/util/db.go]: added

**Pull request merged**